config/.cmdrc_sample.json

```diff
@@ -10,10 +11,12 @@
     "REACT_APP_ENABLE_GQL_MOCK_SERVER": "true"
   },
   "staging": {
+    "REACT_APP_TREND_CHARTS_ENABLED": "false",
     "REACT_APP_API_URL": "https://www.example.com",
     "REACT_APP_UI_URL": "https://www.example.com"
   },
   "prod": {
+    "REACT_APP_TREND_CHARTS_ENABLED": "false",
```

Comment:
Although a feature flag is fine, I don't think that's the intent of EVG-12868. Currently the admin page lets Evergreen administrators turn on and off the trend charts for specific projects.